### PR TITLE
MpxImporter refactor

### PR DESCRIPTION
**Pull request by salvamomo**
## Summary of changes

- Refactor the `MpxImporter` drush command class to use some of the existing Repository and Service classes instead of keeping the update logic in that file, and removed code no longer neededas of that refactor. (Also added a batch_size option to the command).

- Status PR moved to https://github.com/Lullabot/media_mpx/pull/129. This incorporates #127 and #129, so after merging those, the diff here will contain only the refactor.

## Demo / Test Assets

Video showing command usage after the update:
https://take.ms/vNC0t

### CODE REVIEW

**Review by m4olivei — CHANGES_REQUESTED** [review]
> Refactor the MpxImporter drush command class to use some of the existing Repository and Service classes instead of keeping the update logic in that file, and removed code no longer neededas of that refactor. (Also added a batch_size option to the command).

The status page changes are pretty simple and straightforward. This change though seems unrelated and worthy of it's own pull request / review. If it's not too much trouble, I'd like to see it separated.

**Comment by salvamomo** [comment]
That sounds like something I could do! :)

**Comment by salvamomo** [comment]
Moved Status page integration as a separate PR: https://github.com/Lullabot/media_mpx/pull/129.

**Comment by juampynr** [comment]
Merged #129. Thanks for the clip on this one!

**Comment by juampynr** [comment]
cc @edutrul who is working with the importer and @deviantintegral in case he has any additional feedback.

File: src/Commands/MpxImporter.php
Comment (deviantintegral) [inline src/Commands/MpxImporter.php]: This class is already using DI, so use it here too?

**Review by deviantintegral — APPROVED** [review]
One place to use DI, but static review looks great. Feel free to merge when done!

File: src/Commands/MpxImporter.php
Comment (salvamomo) [reply src/Commands/MpxImporter.php]: @deviantintegral great catch. In fact cleaning this up I also noticed 2 other dependencies are not even needed anymore as part of the refactor, and removed those as well \o/. This is much thinner now 🎉

**Comment by salvamomo** [comment]
This is ready for merging. I'll just wait for #127 to be merged before merging this.

**Comment by salvamomo** [comment]
Related PRs are now in place and this is approved. Waiting for tests to pass before merging.

**Pull request merged**
